hooks/class.tx_multicolumn_wizardItemsHook.php

```diff
@@ -22,104 +22,45 @@
  *
  *  This copyright notice MUST APPEAR in all copies of the script!
  ***************************************************************/
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
```

Comment:
I think the whole hook can be removed. The necessary parent information should come from "new content"-link and is passed to the content element automatically (like the colPos information for content elements).